README.md

```diff
@@ -7,7 +7,7 @@ Project DeepSpeech is an open source Speech-To-Text engine, using a model traine
 
 ![Usage](images/usage.gif)
 
-Pre-built binaries that can be used for performing inference with a trained model can be installed with `pip`. Proper setup using virtual environment is recommended and you can find that documented [below](#using-the-python-package).
+Pre-built binaries that can be used for performing inference with a trained model can be installed with `pip3`. Proper setup using virtual environment is recommended and you can find that documented [below](#using-the-python-package).
```

Comment:
"pip3" is OK, but you could also just say "pip", assuming users are on Python 3. Just a suggestion, either are fine.

Reply:
I'm a bit worried about using just `pip` because sometimes python2 can raise it's head. :)